src/api/documents.py

```
@@ -9,10 +9,44 @@
+    try:
+        from services.knowledge_filter_service import KnowledgeFilterService
+
+        svc = KnowledgeFilterService(session_manager)
+        await svc.sync_filters_after_document_rename(
+            old_filename=old_filename,
+            new_filename=new_filename,
+            document_id=document_id,
+            user_id=user_id,
+            jwt_token=jwt_token,
+        )
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)